src/lightning/fabric/strategies/fsdp.py

```diff
@@ -372,20 +385,19 @@ def save_checkpoint(
         from torch.distributed.checkpoint import FileSystemWriter, save_state_dict
         from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
 
-        modules = [module for module in state.values() if isinstance(module, FSDP)]
+        modules = [module for module in state.values() if isinstance(module, Module)]
         if len(modules) == 0:
             raise ValueError(
-                "Could not find a FSDP model in the provided checkpoint state. Please provide the model as"
+                "Could not find a model in the provided checkpoint state. Please provide the model as"
```

Comment:
could we keep the mention of FSDP models specficially?

Reply:
Wouldn't that be misleading? It doesn't match what the code is doing